### PR TITLE
Soildstart/repo readme view

### PR DESCRIPTION
**Pull request by AllStackDev1**
Close: #1391

image

# Create single repo README view

## Background

As a user, by default when I navigate to a single repository I land on the code tab. Here I can see the folders and files found in the root directory of the repo and the name of the root branch. I can also see the repo’s readme file below the folder tree.

## Acceptance

- [x] show root readme.md file with list icon next to the title

### CODE REVIEW

**Review by Megio — APPROVED** [review]
LGTM

File: solidstart-tanstackquery-tailwind-modules/msw/data.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  data: {
+    repository: {
+      readme: {
+        text: 'text',
+      }
+    }
+  }
+};
+
+export const repoInforResponse = {
```
Comment (hdJerry) [inline solidstart-tanstackquery-tailwind-modules/msw/data.ts:11]: why is this data here? It's not part of this ticket

File: solidstart-tanstackquery-tailwind-modules/msw/data.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        nodes: ['topic1', 'topic2', 'topic3']
+      },
+      owner: {
+        orgName: 'tested'
+      }
+    }
+  }
+}
+
+export const repoTreeResponse = {
```
Comment (hdJerry) [inline solidstart-tanstackquery-tailwind-modules/msw/data.ts:41]: why is this data here? It's not part of this ticket

File: solidstart-tanstackquery-tailwind-modules/msw/mockRepoInfo.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { rest } from "msw";
```
Comment (hdJerry) [inline solidstart-tanstackquery-tailwind-modules/msw/mockRepoInfo.ts:1]: why is this data here? It's not part of this ticket
please remove file

File: solidstart-tanstackquery-tailwind-modules/msw/mockRepoTree.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { rest } from "msw";
```
Comment (hdJerry) [inline solidstart-tanstackquery-tailwind-modules/msw/mockRepoTree.ts:1]: why is this data here? It's not part of this ticket
please remove file

**Review by hdJerry — CHANGES_REQUESTED** [review]
There are so many contents that aren't part of the ticket

File: solidstart-tanstackquery-tailwind-modules/msw/mockRepoTree.ts (hunk 4 above)
Comment (AllStackDev1) [reply solidstart-tanstackquery-tailwind-modules/msw/mockRepoTree.ts:1]: I believe testing is part of the completion of a ticket, and also from the last showcases, we have done, we shouldn't wait till we face the issue we faced the last time, which made us change the testing this way. we have to carry alone the lessons of previous showcases to the next one.

File: solidstart-tanstackquery-tailwind-modules/msw/mockRepoInfo.ts (hunk 3 above)
Comment (AllStackDev1) [reply solidstart-tanstackquery-tailwind-modules/msw/mockRepoInfo.ts:1]: I shouldn't have done this.

File: solidstart-tanstackquery-tailwind-modules/msw/data.ts (hunk 2 above)
Comment (AllStackDev1) [reply solidstart-tanstackquery-tailwind-modules/msw/data.ts:41]: I shouldn't have done this.

File: solidstart-tanstackquery-tailwind-modules/msw/data.ts (hunk 1 above)
Comment (AllStackDev1) [reply solidstart-tanstackquery-tailwind-modules/msw/data.ts:11]: I shouldn't have done this.

**Pull request merged**